src/app/zap-templates/zcl/data-model/chip/joint-fabric-pki-cluster.xml

```diff
@@ -0,0 +1,89 @@
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="ICACSigningRequestStatusEnum" type="enum8">
+    <cluster code="0x0753"/>
+    <item name="OK" value="0x00"/>
+    <item name="InvalidIcaCsrFormat" value="0x01"/>
+    <item name="InvalidIcaCsrSignature" value="0x02"/>
+    <item name="FailedDclVendorIdValidation" value="0x03"/>
+    <item name="NotAnIcac" value="0x04"/>
+    <item name="BusyAnchorTransfer" value="0x05"/>
+    <item name="IcaCsrSigningFailed" value="0x06"/>
+    <item name="IcaCsrRequestNoUserConsent" value="0x07"/>
+  </enum>
+
+  <enum name="TransferAnchorResponseStatusEnum" type="enum8">
+    <cluster code="0x0753"/>
+    <item name="OK" value="0x00"/>
+    <item name="TransferAnchorStatusDatastoreBusy" value="0x01"/>
+    <item name="TransferAnchorStatusNoUserConsent" value="0x02"/>
+  </enum>
+
+  <cluster>
+    <domain>General</domain>
+    <name>Joint Fabric Pki</name>
+    <code>0x0753</code>
+    <define>JOINT_FABRIC_PKI_CLUSTER</define>
+    <description>Joint Fabric Pki Cluster.</description>
+
+    <command source="client" code="0x00" name="ICACSRRequest" response="ICACSRResponse" optional="false">
+      <description>Client requests Server's ICA CSR.</description>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="server" code="0x01" name="ICACSRResponse" optional="false">
+      <description>Server requests ICAC to be signed by Client's PKI.</description>
+      <arg name="NOCIssuerCSR" type="octet_string" length="400"/>
+    </command>
+
+    <command source="client" code="0x02" name="AddICAC" response="AddICACResponse" optional="false">
+      <description>Joint Fabric ICAC generated.</description>
+      <arg name="StatusCode" type="ICACSigningRequestStatusEnum"/>
+      <arg name="NOCIssuerCert" type="octet_string" length="400"/>
+      <arg name="NodeId" type="node_id"/>
+      <arg name="FabricId" type="fabric_id"/>
+      <arg name="AdminVendorId" type="vendor_id"/>
+      <arg name="CaseAdminSubject" type="int64u"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="server" code="0x03" name="AddICACResponse" optional="false">
+      <description>Response to AddICAC command.</description>
+      <arg name="StatusCode" type="ICACSigningRequestStatusEnum"/>
+      <arg name="FabricIndex" type="fabric_idx" optional="true"/>
+      <arg name="DebugText" type="char_string" optional="true" length="128"/>
+    </command>
```

Comment:
Not compliant with ICACResponse from https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/10743